### ISSUE TITLE
Can't join to project after clicked 'Join channel'

### ISSUE THREAD

**Issue by hzhshok** [AgoraIO-Community/Agora-Unreal-SDK#12]
It showed some errors on log console without any video/audio display on UE running after i ran UE project, and not join to the specific project(on Web client）created on project management page.

The log showed:

Voice level:

**Video level:**
LogTemp: Warning: UVideoViewUserWidget OnJoinButtonClick ======
LogTemp: Warning: registerVideoFrameObserver : 0
LogTemp: Warning: JoinChannel ret: 0
LogTemp: Warning: onJoinChannelSuccess channel: 整瑳Ā, uid: 2121241643, sdk ver: ⸳⸵0
LogTemp: Warning: onError err: 1502, msg: (null)
LogTemp: Warning: onError err: 1003, msg: (null)
LogTemp: Warning: UVideoViewUserWidget clean =====onJoinChannelSuccess ======
LogTemp: Warning: VideoFrameObserver::addUserInfo uid: 0

I also check error code from sdk include files, those codes represent, but don't know why?
  ERR_VDM_WIN_DEVICE_IN_USE = 1502,
  ERR_START_CAMERA = 1003,

**Hardware**: x64 PC(intel core i9...).
**Software**: UE4.26(also tried UE5.0EA).
**Configuration**:
a. Have added AgoraPlugin into API_Example\Plugins.
b. AgoraPlugin is also able to pass compilation.
c. Generate temporal  token for channel 'test', and change the channel name and token before the following line: 
        tokenStr = "..."; // generated temporal  token;
           const char* CHANNEL_NAME = "test"; // ?? don't know if token name is acscii char??
        auto ret = _rtcEnginePtr->joinChannel(tokenStr, channelName, "", 0);

Regards
Zhansheng

**Comment by hzhshok** [comment]
Supply something:
a. Have add AgoraPlugin into API_Example\Plugins. 
b. AgoraPlugin  is also able to pass compilation.

**Comment by HugoChaan** [comment]
Hello, In Voice level, Error code returns 110 means you have used an AppId with a token, you could use an AppId without a
 token instead. In Video level,  Error code returns 1003 means your camera has been occupied.

**Comment by hzhshok** [comment]
Thanks! 
I made channel token into UE project after reading doc, and also close invitation page(maybe opened web client for camera/voice),  the UE could show video window normally, but i did not test the multiple for one channel, and later i will check this functionality about chat with multiple site.

Anyway, i shall close this issue first since that error showed it was caused by configuration and operation.

Regards
Zhansheng HAI
